src/plugins/hetero/src/config.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "config.hpp"
+
+// #include <cpp_interfaces/interface/ie_internal_plugin_config.hpp>
+#include <ie_plugin_config.hpp>  // TODO vurusovs create file with custom properties
+
+#include "openvino/runtime/internal_properties.hpp"
+#include "openvino/runtime/properties.hpp"
+
+using namespace ov::hetero;
+
+Configuration::Configuration() {}
+
+Configuration::Configuration(ov::AnyMap& config, const Configuration& defaultCfg, bool throwOnUnsupported) {
+    *this = defaultCfg;
+
+    for (auto conf_it = config.begin(); conf_it != config.end();) {
+        auto it = conf_it++;
```

Comment:
Why not:
```suggestion
    for (const auto& it = config.begin(); t != config.end(); it++) {
```

Reply:
Because I want to erase all "hetero" properties from `config` and leave only "device" properties, I can't iterate as is